rulebook.tex

```diff
@@ -2121,63 +2145,71 @@ \subsection{Order Schedule}
 production plan. However, each game will use a new randomized order
 schedule.
 
-\subsection{Scheduled Machine Downtime --- Main Track}
+\subsubsection{Scheduled Machine Downtime --- Main Track}
 There will be exactly two occurrences of scheduled machine downtime for each
 game in the main track according to the rules in \refsec{sec:out-of-order}.
 
-\subsection{Tournament Phases}
+\subsubsection{Overview --- Main Track}
 \label{sec:tournament-phases}
-There will be three stages in the main competition, a round-robin
+There will be three stages in the main track, a (optional) round-robin
 phase for all participating teams, playoffs for the best four teams
```

Comment:
Since there is also playoffs for up to 6 teams as well, this should be clarified to avoid misconceptions.

Reply:
true, fixed this.